src/Selenium2Driver.php

```diff
@@ -189,14 +189,7 @@ public static function getDefaultCapabilities()
     {
         return array(
             'browserName'       => 'firefox',
-            'version'           => '9',
```

Comment:
Can we make the default `version` the empty string, which signifies the "latest" available version of the browser. I think it's a better default than forcing users to specify the browser version explicitly (and it would make the change backwards compatible). What do you think?